golem_messages/factories/tasks.py

```diff
@@ -23,9 +24,10 @@ class Meta:
     node_name = factory.Faker('name')
     provider_public_key = factory.LazyFunction(
         lambda: encode_key_id(cryptography.ECCx(None).raw_pubkey))
-    provider_ethereum_public_key = factory.SelfAttribute(
-        'provider_public_key'
-    )
+    provider_ethereum_address = factory.LazyFunction(
```

Comment:
@tworec please make sure it uses `provider_public_key` as the basis of the address and not a completely different key

Reply:
this factory was wrong: `provider_public_key` has nothing to do with `provider_ethereum_public_key`, therefore  `provider_ethereum_address` shouldn't base on it also

Reply:
@tworec I'll insist here ...

Reply:
ok